kiosk/package.json

```diff
@@ -11,8 +11,10 @@
     "@types/node": "^16.11.33",
     "@types/react": "^18.0.9",
     "@types/react-dom": "^18.0.3",
+    "qrcode.react": "^3.1.0",
     "react": "^18.1.0",
```

Comment:
Could consider dropping down react version in kiosk app to whatever version is needed for things to work without the flag, but looking at it seems like it'll probably end up fine. definitely needs a note in a readme somewhere though, I would be very hesitant if I came to rebuild this & all of a sudden a flag I hadn't seen before was necessary.

Reply:
Yeah, I was thinking about this, too. Especially since it allows react 17. I'll see if I can change the react version to solve the problem.

Reply:
@jwunderl Just for an update, I realized why this wasn't working now but was working before (there are other dependencies that strictly require an earlier version of React in their peer dependencies but weren't throwing errors). It's because npm in newer versions is checking and more strictly enforcing peer dependencies. Older npm versions don't do this which is also why the build didn't fail because our build in arcade is using node 14.

Reply:
In my dev environment, I use node 16.17.0, but once I changed my environment to use 16.12.0, you can npm install fine.